dev/io.openliberty.java.internal_fat/bnd.bnd

```diff
@@ -18,7 +18,7 @@ fat.project: true
 
 # To define a global minimum java level for the FAT, use the following property.
 # If unspecified, the default value is ${javac.source}
-fat.minimum.java.level: 17
+#fat.minimum.java.level: 17
```

Comment:
Specifying this value breaks this test when you run it locally since the logic in launch.xml detects if the `fat.minimum.java.level` specified here is greater than the running JVM.  If so, launch.xml tries to perform these 2 commands:
```
<var name="test.bucket.class" unset="true" />
<var name="test.bucket.name" unset="true" />
```
but the `var` type which is a tool not available to local runs (but is on the test servers).